monkey/infection_monkey/master/propagator.py

```diff
@@ -4,7 +4,7 @@
 from queue import Queue
 from typing import List, Mapping, MutableMapping, Sequence
 
-from agentpluginapi import ExploiterResult, PingScanData, PortScanData, TargetHost
+from agentpluginapi import ExploiterResult, FingerprintData, PingScanData, PortScanData, TargetHost
```

Comment:
Is this under the line length limit?

Reply:
Black is formatting it like this, so I guess it is.